website/addons/googledrive/views/config.py

```diff
@@ -34,41 +35,65 @@ def googledrive_config_get(node_addon, auth, **kwargs):
 def googledrive_config_put(node_addon, auth, **kwargs):
     """View for changing a node's linked Google Drive folder/file."""
     selected = request.get_json().get('selected')
-    node_addon.set_folder(selected, auth=auth)
+    node_addon.set_target_folder(selected, auth=auth)
     node_addon.save()
     return {
         'result': {
             'folder': {
                 'name': selected['path'],
             },
-            'urls': serialize_urls(node_addon),
+            'urls': GoogleDriveSerializer(
+                node_settings=node_addon,
+                user_settings=auth.user.get_addon('googledrive')
+            ).serialized_urls
         },
         'message': 'Successfully updated settings.',
     }
 
-
 @must_be_logged_in
 def googledrive_user_config_get(auth, **kwargs):
```

Comment:
The route for this view function was removed, so this should be taken out as well.